rspace/src/main/scala/coop/rchain/rspace/HotStore.scala

```diff
@@ -72,15 +74,7 @@ private class InMemHotStore[F[_]: Sync, C, P, A, K](
       cached <- internalGetContinuations(channels)
       state  <- S.read
       result = state.installedContinuations.get(channels) ++: cached
-      res <- result
-              .map(
-                wk =>
-                  roundTrip[F, K](wk.continuation).map { copied =>
-                    wk.copy(continuation = copied)
-                  }
-              )
-              .sequence
-    } yield res
+    } yield result
```

Comment:
please also remove the now-unused roundtrip method though

Reply:
Done